scripts/commit_classifier.py

```diff
@@ -377,6 +373,8 @@ def load_user(phid):
                     check=True,
                     cwd=self.git_repo_dir,
                 )
+        latest_rev = repository.get_revs(hg, f"-{len(stack)}")
```

Comment:
You're setting this variable here, but trying to use it earlier :smile: 
Maybe this is not an easy enough task to start with, I'd suggest working on some personal projects first to get some initial experience with programming.